internal/restore/volumesnapshot_action.go

```diff
@@ -93,6 +93,12 @@ func (p *VolumeSnapshotRestoreItemAction) Execute(input *velero.RestoreItemActio
 			deletionPolicy = string(snapshotv1api.VolumeSnapshotContentRetain)
 		}
 
+		if deletionPolicy == string(snapshotv1api.VolumeSnapshotContentDelete) {
```

Comment:
why not just ignore the annotation and set the deletion policy always to retain, and print a explicit message.

Reply:
Reasonable. My thought is limited the log print numbers.